src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -44,18 +45,20 @@
   private static final String APPLICATION_NAME = "SAY";
 
   /**
-   * Applies parameters to the YouTube API, then extracts comments. URL is the only true variable;
-   * for this application we will always want order to be relevance and max results to be 100, the
-   * API's arbitrary limit
+   * Retrieves comments from designated URL, passes them off to CommentAnalysis object to be wrapped
+   * into Statistics object, then writes the Statistics object to the frontend.
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
     try {
       String url = request.getParameter(URL_PARAMETER);
       CommentThreadListResponse commentResponse = generateYouTubeRequest(url).execute();
-      // TODO: input commentResponse into commentAnalysis class to get useful data
-      Statistics statistics = new Statistics(new ArrayList<Double>());
+
```

Comment:
extra line?

Reply:
That's odd; there doesn't seem to be an extra line on my side. I split up the functionality of doGet with line breaks, so each step is seperated by some space.